test/bdd/bddtests_test.go

```diff
@@ -30,8 +34,38 @@ func TestMain(m *testing.M) {
 
 	status := godog.RunWithOptions("godogs", func(s *godog.Suite) {
 		s.BeforeSuite(func() {
+
+			if os.Getenv("DISABLE_COMPOSITION") != "true" {
+
+				// Need a unique name, but docker does not allow '-' in names
+				composeProjectName := strings.Replace(generateUUID(), "-", "", -1)
+				newComposition, err := NewComposition(composeProjectName, "docker-compose.yml", "./fixtures")
+				if err != nil {
+					panic(fmt.Sprintf("Error composing system in BDD context: %s", err))
+				}
+
+				composition = newComposition
+
+				fmt.Println("docker-compose up ... waiting for peer to start ...")
+				testSleep := 5
+				if os.Getenv("TEST_SLEEP") != "" {
+					testSleep, _ = strconv.Atoi(os.Getenv("TEST_SLEEP"))
+				}
+				fmt.Printf("*** testSleep=%d", testSleep)
```

Comment:
what's TEST_SLEEP?

Reply:
TEST_SLEEP is used to wait until all docker containers are started